vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,16 +111,82 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
     }
+    
+    /**
+     * To create or update the default link images.
+     * 
+     * @param exhibitID - the ID of the exhibition to update
+     * @param spaceLinkImage - the MultipartFile representing the space link image to be uploaded
+     * @param moduleLinkImage - the MultipartFile representing the module link image to be uploaded
+     * @param externalLinkImage - - the MultipartFile representing the external link image to be uploaded
+     * @param attributes - - the RedirectAttributes object to add flash attributes
+     * @return - a RedirectView to the exhibition configuration page
+     * @throws IOException if an input or output error occurs
+     */
+    @RequestMapping(value = "/staff/exhibit/config/link/defaultImage", method = RequestMethod.POST)
+    public RedirectView createOrUpdateLinkImage(HttpServletRequest request,
+            @RequestParam(required = false, name = "exhibitionParam") String exhibitID,
+            @RequestParam(name = "spaceLinkImage", required = false) MultipartFile spaceLinkImage,
+            @RequestParam(name = "moduleLinkImage", required = false) MultipartFile moduleLinkImage,
+            @RequestParam(name = "externalLinkImage", required = false) MultipartFile externalLinkImage,
+            RedirectAttributes attributes) throws IOException {
+        Exhibition exhibition;
+        if (exhibitID == null || exhibitID.isEmpty()) {
+            exhibition = (Exhibition) exhibitFactory.createExhibition();
+        } else {
+            exhibition = (Exhibition) exhibitManager.getExhibitionById(exhibitID);
+        }
+        
+        if(spaceLinkImage != null) {
+            IVSImage spaceDefaultImage = imageService.storeImage(spaceLinkImage.getBytes(), spaceLinkImage.getOriginalFilename()); 
+            exhibition.setSpaceLinkDefaultImage(spaceDefaultImage);
+        }
+        else if(moduleLinkImage!=null) {
+            IVSImage moduleDefaultImage = imageService.storeImage(moduleLinkImage.getBytes(), moduleLinkImage.getOriginalFilename());            
+            exhibition.setModuleLinkDefaultImage(moduleDefaultImage);
+        }
+        else if(externalLinkImage!=null) {
+            IVSImage externalLinkDefaultImage = imageService.storeImage(externalLinkImage.getBytes(), externalLinkImage.getOriginalFilename());         
+            exhibition.setExternalLinkDefaultImage(externalLinkDefaultImage);
+        }
+        else {
+            attributes.addAttribute("alertType", "danger");
+            attributes.addAttribute("showAlert", "true");
+            attributes.addAttribute("message", "Please select an image");
+            return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+        }
+        
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
+        attributes.addAllAttributes(addSuccessAttributes(exhibition));
+
+        return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
```

Comment:
there should be three different forms to submit the default images; each should call this function with a parameter defining which default image to set; and then map the parameter to a function that sets the default on the image at exhibition.